File: app/src/main/java/com/example/irkpo_management/models/DaySchedule.java
package com.example.irkpo_management.models;

import java.util.List;

public class DaySchedule {
    private String dayOfWeek;
    private int weekType;
    private List<LessonIndex> lessonIndexes; // Уроки на этот день

    // Геттеры
    public String getDayOfWeek() {
        return dayOfWeek;
    }

    public int getWeekType() {
        return weekType;
    }

    public List<LessonIndex> getLessonIndexes() {
        return lessonIndexes;
    }
}

File: app/src/main/java/com/example/irkpo_management/models/DaySchedule.kt
package com.example.irkpo_management.models

class DaySchedule {
    // Геттеры
    @JvmField
    val dayOfWeek: String? = null
    @JvmField
    val weekType: Int = 0
    @JvmField
    val lessonIndexes: MutableList<LessonIndex?>? = null // Уроки на этот день
}
